laplace/baselaplace.py

```diff
@@ -1303,6 +1303,10 @@ def posterior_precision(self):
         precision : `laplace.utils.matrix.KronDecomposed`
         """
         self._check_H_init()
+
+        if self.H_facs is None:
+            raise AttributeError('Laplace not fitted. Run fit() first.')
```

Comment:
Should the `_check_H_init` method not be overridden for `KronLaplace` (with these lines)?